packages/gatsby-theme-apollo-docs/src/components/code-block.js

```diff
@@ -21,7 +21,7 @@ const Header = styled.div({
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'flex-end',
-  padding: 10,
+  padding: 4,
```

Comment:
Reduce the height of codeblock headers to increase information density slightly
![Uploading Screen Shot 2021-03-24 at 4.33.15 PM.png…]()